explanation/README.md

```diff
@@ -1,117 +1,45 @@
 
 The figure below again shows SN 2019yvq, the running median used to determine the time of maximum and the baseline region used to determine `C`.
 
 <img src="./../images/baseline_max.jpg" raw=True>
 
 ### Scaling the uncertainties
 
-Following the recommendation in the [fps
-documentation](http://web.ipac.caltech.edu/staff/fmasci/ztf/forcedphot.pdf),
-we re-scale the uncertainties of all observations based on the chi-squared
-distribution of observations in the baseline region. In brief, flux
-measurements in an "empty" patch of sky should follow a gaussian distribution
-and "excess" scatter points to some systematic uncertainty that has not been
-properly quantified (subtractions near galaxy nuclei often exhibit such excess
-scatter). If the chi-squared distribution is >1, then the uncertainties *of
-all observations* are multiplied by the square root of the reduced chi squared
-in the baseline region (see also [Yao et al.
-(2019)](http://dx.doi.org/10.3847/1538-4357/ab4cf5)).
+Following the recommendation in the [fps documentation](http://web.ipac.caltech.edu/staff/fmasci/ztf/forcedphot.pdf), we re-scale the uncertainties of all observations based on the chi-squared distribution of observations in the baseline region. In brief, flux measurements in an "empty" patch of sky should follow a Gaussian distribution and "excess" scatter points to some systematic uncertainty that has not been properly quantified (subtractions near galaxy nuclei often exhibit such excess scatter). If the mean of the chi-squared distribution is < 1.5, then the uncertainties *of all observations* are multiplied by the square root of the median chi-squared distribution in the baseline region. Otherwise, if there are "good observations" (`infobits` = 0, see above), a [SuperSmoother](https://github.com/jakevdp/supersmoother/) model is used to obtain a smooth fit of chi-squared distribution as function of flux of these "good observations". The chi-squared distribution for the "bad observations" (`infobits` > 0) are obtained by interpolating the chi-squared distributions of the "good observations". In case the model fails, 1000s rolling medians on the "good observations" are used for the interpolation. The uncertainties *of all observations* are then multiplied by the square root of all the interpolated chi-squared distributions (see also [Yao et al. (2019)](http://dx.doi.org/10.3847/1538-4357/ab4cf5)).
```

Comment:
This text is a little unclear at the moment. It might also help if we create a plot or two demonstrating what this procedure looks like.